libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/button/MainActionButton.kt

```diff
@@ -58,8 +57,8 @@ fun MainActionButton(
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
         Icon(
+            contentDescription = null,
```

Comment:
Shouldn't we move this content description to the component itself with a `semantics` modifier?

Reply:
Ah, I didn't realise the title was used below, sorry.